corehq/apps/reports/views.py

```diff
@@ -789,14 +795,14 @@ def scheduled_report_form(self):
         form.fields['config_ids'].choices = self.config_choices
         form.fields['recipient_emails'].choices = [(e, e) for e in web_user_emails]
 
-        form.fields['hour'].help_text = "This scheduled report's timezone is %s (%s GMT)" % \
+        form.fields['hour'].help_text = _("This scheduled report's timezone is %s (%s GMT)" % \
                                         (Domain.get_by_name(self.domain)['default_timezone'],
-                                        get_timezone_difference(self.domain)[:3] + ':'
-                                        + get_timezone_difference(self.domain)[3:])
-        form.fields['stop_hour'].help_text = "This scheduled report's timezone is %s (%s GMT)" % \
+                                        get_timezone_difference(self.domain).hours + ':'
+                                        + get_timezone_difference(self.domain).minutes))
+        form.fields['stop_hour'].help_text = _("This scheduled report's timezone is %s (%s GMT)" % \
                                         (Domain.get_by_name(self.domain)['default_timezone'],
-                                        get_timezone_difference(self.domain)[:3] + ':'
-                                        + get_timezone_difference(self.domain)[3:])
+                                        get_timezone_difference(self.domain).hours + ':'
+                                        + get_timezone_difference(self.domain).minutes))
```

Comment:
With the `__str__` suggestion above this would become:
```suggestion
                                        get_timezone_difference(self.domain))
```